tests/baselines/reference/typeGuardsWithInstanceOfByConstructorSignature.errors.txt

```diff
@@ -84,11 +82,9 @@ tests/cases/conformance/expressions/typeGuards/typeGuardsWithInstanceOfByConstru
         obj5.foo;
         obj5.c;
         obj5.bar1;
-             ~~~~
-!!! error TS2339: Property 'bar1' does not exist on type 'C1 | C2'.

```

Comment:
I believe this is an improvement -- `obj5` was of type `C1 | A`, so a narrowing to `instanceof C` should not add `C2` back into the mix.